.github/workflows/next-on-call.yml

```diff
@@ -21,7 +21,7 @@ jobs:
 
       - name: Run CLI
         id: cli
-        run: echo "::set-output name=payload::$(artsy scheduled:next-on-call)"
+        run: echo "PAYLOAD=$(artsy scheduled:next-on-call)" >> $GITHUB_OUTPUT
```

Comment:
**suggestion**: Might be a good idea to double quote this environment variable name, just in case something changes internally that includes an unexpected space.

```suggestion
        run: echo "PAYLOAD=$(artsy scheduled:next-on-call)" >> "$GITHUB_OUTPUT"
```

Reply:
Done in https://github.com/artsy/joule/pull/141/commits/9706b677441e49393e9a098f9e75aef3ed63cceb